test/spec/smoke_spec.rb

```diff
@@ -1553,8 +1553,14 @@
   end
 
   it "should search by project through URL", bin2: true do
-    api_create_team_project_and_claim_and_redirect_to_media_page
-    @driver.navigate.to @config['self_url'] + '/' + get_team + '/all-items/%7B"projects"%3A%5B' + get_project + '%5D%7D'
+    # TODO: Sawy - review
```

Comment:
Review what?

Reply:
was added to flag failed test, removed it